_episodes/04-history.md

```diff
@@ -250,25 +294,21 @@ $ git status
 {: .bash}
 
 ~~~
-# On branch master
-Changes to be committed:
-  (use "git reset HEAD <file>..." to unstage)
-# Changes not staged for commit:
-#   (use "git add <file>..." to update what will be committed)
-#   (use "git checkout -- <file>..." to discard changes in working directory)
-#
-#	modified:   mars.txt
-#
-no changes added to commit (use "git add" and/or "git commit -a")
+On branch master
```

Comment:
Won't this say something like 'detached HEAD state', instead of `master`?

Reply:
We did a checkout of README, not a checkout of the commit w/ no referenced file. I can go back and check my history, but I believe this is right.

Reply:
Oh, I see.